Description: Correct pod NAME issue in LedgerSMB/User.pm
 Has the module name but not the dash plus short description in the NAME entry.
Bug: http://sourceforge.net/tracker/?func=detail&aid=3472647&group_id=175965&atid=1005191
Author: Robert James Clay <jame@rocasa.us>

--- a/LedgerSMB/User.pm
+++ b/LedgerSMB/User.pm
@@ -1,7 +1,7 @@
 
 =head1 NAME
 
-LedgerSMB::User
+LedgerSMB::User - Provides user support and database managment functions.
 
 =head1 SYNOPSIS
 
